### ISSUE TITLE
Add "main" element to pages - a11y

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#120]
Document must have a `<main>` element landmark.

**Comment by lhenze** [comment]
From NYU A11y review - 

The homepage is missing a "main" landmark. This is important to identify the main part of the page as well as to ensure a "skip to main' bypass block has a destination. Also, recommend that a "search" landmark be added to the search box in the nav area (as it is in the banner of all search results pages):

<div class="ui fluid big action input" role="search">
   <input type="text" name="q" class="form-control" placeholder="Search">
   <button type="submit" class="ui icon search button"><i class="search icon"></i></button>
</div>

**Comment by lhenze** [comment]
In core InvenioRDM, all pages now have a main element. (And also "skip to main") function.
